buildstockbatch/aws/aws.py

```diff
@@ -241,23 +237,26 @@ def create_vpc(self):
 
         logger.info("Private subnet created.")
 
-        self.ec2.create_tags(
+        backoff(
+            self.ec2.create_tags,
             Resources=[self.priv_vpc_subnet_id_1],
-            Tags=[{"Key": "Name", "Value": self.job_identifier}],
+            Tags=self.get_tags_uppercase(Name=self.job_identifier),
         )
 
-        self.ec2.create_tags(
+        backoff(
```

Comment:
I see this pattern a lot.  Maybe create a function?
```suggestion
        def create_tags(self, resource):
            backoff(self.ec2.create_tags, Resources=[resource], Tags=self.get_tags_uppercase(Name=self.job_identifier)
```

Reply:
It is already pretty abstracted with the `backoff` function. I'd like to keep it general instead of creating one just for the ec2.create_tags function.